CMakeLists.txt

```diff
@@ -122,11 +90,13 @@ find_library(libuv_shared NAMES uv libuv PATHS ${LIBUV_DEPS_DIR}/src PATH_SUFFIX
 # Referenced directories and targets
 #
 
-if(${DOXYGEN_FOUND})
-    add_subdirectory(${PROJECT_DOCS_DIR})
-endif(${DOXYGEN_FOUND})
+if(DOXYGEN_FOUND)
+    add_subdirectory(docs)
+endif()
 
-add_subdirectory(${PROJECT_TEST_DIR})
+if(BUILD_TESTING)
```

Comment:
Done. Introducing `BUILD_TESTING` without `include(CTest)`.
Moved `enable_testing()` inside the `if` block.